### PR TITLE
chore: add grpc_server_example, grpc-web-server and node_modules to .gitignore

### PR DESCRIPTION
**Pull request by vyvyan2010**
### Description

add grpc_server_example, grpc-web-server and node_modules to .gitignore

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -77,6 +77,10 @@ t/fuzzing/__pycache__/
 boofuzz-results/
 *.pyc
 *.wasm
+grpc_server_example
```
Comment (spacewander) [inline .gitignore:80]: We should not add directory grpc_server_example into gitignore?

Comment (vyvyan2010) [reply .gitignore:80]: It's not directory, just grpc_server_example binary executable file
image

Comment (spacewander) [reply .gitignore:80]: It will affect any new file in this directory?

Comment (vyvyan2010) [reply .gitignore]: sorry, i have updated the .gitignore file, please check.

**Review by spacewander — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**